plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -44,6 +42,20 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	SchedulingAlgorithmPredicateEvaluationDuration = prometheus.NewGauge(
+		prometheus.GaugeOpts{
```

Comment:
This is something I think we're not doing anywhere else, as having this graph requires running prometheus server, or something else that scrapes timeseries. As all our SLIs speak about percentiles we generally use Histogram for any performance-related metrics.